frontend/ui/src/pages/Channels/Facebook/RequirementsDialog.module.scss

```diff
@@ -84,3 +84,34 @@
   cursor: pointer;
   outline: none;
 }
+.inviteWrapper {
```

Comment:
as you added the Twilio requirementsDialog inside the Facebook/RequirementsDialog your css changes will affect the requirementsDialog of the facebook flow.